garak/generators/openai.py

```diff
@@ -98,13 +98,11 @@ class OpenAICompatible(Generator):
         "frequency_penalty": 0.0,
         "presence_penalty": 0.0,
         "stop": ["#", ";"],
+        "blocked_params": set(),
+        "generations": 10,
```

Comment:
```suggestion
```
Remove value not consumed and defer `generations ` as provided by `Generator.DEFAULT_PARAMS`